tests/main/views/test_buyers.py

```diff
@@ -3617,6 +3617,43 @@ def test_cancel_brief_200s_with_correct_default_content(self):
         submit_button = document.xpath('//input[@class="button-save" and @value="Update requirement"]')
         assert len(submit_button) == 1
 
+        expected_previous_page_link_text = 'Previous page'
+        expected_previous_page_link_url = (
+            '/buyers/frameworks/digital-outcomes-and-specialists-2/requirements/digital-outcomes/1234'
+        )
+
+        assert (
+            document.xpath("//a[text()='{}']/@href".format(expected_previous_page_link_text))[0] ==
+            expected_previous_page_link_url
+        )
```

Comment:
Hmm... rather than searching for the expected link text, could we instead assert that there is a single link inside the `secondary-action-link` div (we should only have one of those per page, right?) and that this link has the expected text/url? (Same for the test below)

Reply:
So I did this because I don't want changes in the frontend toolkit or refactoring of scss class names to fail our tests. 
If I managed to put the `Previous page` link in the correct position on the page _without_ using `secondary-action-link` for whatever reason the requirements of the ticket would still be satisfied and the tests should still pass.
If you feel strongly I'll change it.

Reply:
Yeah, that makes sense, happy for it to stay then.